src/saving/serializers/ThriveJsonConverter.cs

```diff
@@ -978,7 +978,7 @@ private void UpdateObjectReference(JsonSerializer serializer, object oldReferenc
 /// <summary>
 ///   When a class has this attribute DefaultThriveJSONConverter is used to serialize it
 /// </summary>
-[AttributeUsage(AttributeTargets.Class)]
+[AttributeUsage(AttributeTargets.Class | AttributeTargets.Interface)]
```

Comment:
I'm kind of suspicious about allowing this on interfaces...

Reply:
Wasn't this the final conclusion on the way how to go about it from the last time this is discussed?

Reply:
https://discord.com/channels/464846402732687391/464846955873304587/969962030977122355, or probably I'm misunderstanding things again on the approach?

Reply:
Okay, my memory hasn't fully returned, but maybe the advice should be that when you use dynamic type, the base type that is used in the containing object (even if it is an interface) needs to specify the thrive serializer. I think if that's the case then this change and the styleguide change (once the explanation is improved), is good.

Reply:
I'll mark this conversation as resolved but leave the one in the styleguide which is where the "fix" is needed for this.